config/reports/test-report.jinja2

```diff
@@ -1,16 +1,21 @@
 {# -*- mode: Python -*- -#}
 {# SPDX-License-Identifier: LGPL-2.1-or-later -#}
 
-{{ root.revision.tree }}/{{ root.revision.branch }} {{ '(Unknown)' if root.revision.describe is none else root.revision.describe}}: {{ total_runs }} runs, {{ total_failures}} failures
+{{ subject_str }}
+
+Tests Summary
+-------------
+
+Build details:
+
+Tree:     {{ root.revision.tree }}
+Branch:   {{ root.revision.branch }}
+Describe: {{ '(Unknown)' if root.revision.describe is none else root.revision.describe}}
```

Comment:
Out of interest, do we actually have cases where `describe` is `None`?

Reply:
So please drop the conditional logic

Reply:
ops, I did the change but forgot to push..updated now